### PR TITLE
add certificate command tests

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
#### What are the relevant tickets?
https://github.com/mitodl/bootcamp-ecommerce/issues/1115

#### What's this PR do?
Adds certificate command tests

#### How should this be manually tested?
Simply run the tests with pytest

### CODE REVIEW

File: klasses/commands_test.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        ),
+        (None, None),
+    ],
+)
+def test_manage_certificates_command(mocker, method, command_kwargs):
+    """Verify manage_certificate command execution flow"""
+
+    def run_command(command, *args, **kwargs):
+        """Run management command"""
+        out = StringIO()
```
Comment (umarmughal824) [inline klasses/commands_test.py:41]:
```suggestion
        output = StringIO()
```

Comment (umarmughal824) [reply klasses/commands_test.py:41]: also, change its associated uses

File: klasses/commands_test.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        call_command(command, *args, stdout=out, stderr=StringIO(), **kwargs)
+        return out.getvalue()
+
+    if method:
+        method_patch = mocker.patch(method, return_value={"updated": True, "msg": ""})
```
Comment (umarmughal824) [inline klasses/commands_test.py:46]:
```suggestion
        patched_method = mocker.patch(method, return_value={"updated": True, "msg": ""})
```

Comment (umarmughal824) [reply klasses/commands_test.py:46]: also, change its associated uses

**Review by umarmughal824 — CHANGES_REQUESTED** [review]
some minor changes

File: klasses/commands_test.py (hunk 2 above)
Comment (HamzaIbnFarooq) [reply klasses/commands_test.py]: updated.

File: klasses/commands_test.py (hunk 1 above)
Comment (HamzaIbnFarooq) [reply klasses/commands_test.py]: updated.

**Review by umarmughal824 — APPROVED** [review]
👍

**Pull request merged**